scheduler/rank.go

```diff
@@ -195,6 +199,16 @@ OUTER:
 				DiskMB: int64(iter.taskGroup.EphemeralDisk.SizeMB),
 			},
 		}
+
```

Comment:
I think this method has to be refactored. It will get even more complex once devices are considered. I had a hard to following the code and I know it quite well